chain/rand/fake.go

```diff
@@ -0,0 +1,29 @@
+package rand
+
+import (
+	"context"
+	"math/rand"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/crypto"
+	"github.com/filecoin-project/lotus/chain/vm"
+)
+
+type fakeRand struct{}
+
+func NewFakeRand() vm.Rand {
```

Comment:
why not fakeRand is returned?

Reply:
This was a left-over for some testing I was doing, I removed the function. Thanks!